modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,39 @@
+[id="enabling-and-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+You can enable the Bulk Import feature for users and give them the necessary permissions to access it.
```

Comment:
Please add some spaces here, as it's not rendering correctly.

![image](https://github.com/user-attachments/assets/d3e8ee0e-223f-43b2-9bc2-4642bd708073)

Reply:
```suggestion
You can enable the Bulk Import feature for users and give them the necessary permissions to access it.

```